engine/src/main/resources/org/hibernate/validator/ValidationMessages_de.properties

```diff
@@ -1,40 +1,53 @@
 
-org.hibernate.validator.constraints.CreditCardNumber.message = ung\u00FCltige Kreditkartennummer
-org.hibernate.validator.constraints.Currency.message         = ung\u00FCltige W\u00E4hrung (erlaubte Werte: {value})
-org.hibernate.validator.constraints.EAN.message              = ung\u00FCltiger {type} Barcode
-org.hibernate.validator.constraints.Email.message            = keine g\u00FCltige E-Mail-Adresse
-org.hibernate.validator.constraints.Length.message           = muss zwischen {min} und {max} Zeichen lang sein
-org.hibernate.validator.constraints.CodePointLength.message  = muss zwischen {min} und {max} Zeichen lang sein
-org.hibernate.validator.constraints.NotBlank.message         = darf nicht leer sein
-org.hibernate.validator.constraints.NotEmpty.message         = darf nicht leer sein
-org.hibernate.validator.constraints.ParametersScriptAssert.message = Skriptausdruck "{script}" gab nicht true zur\u00FCck
-org.hibernate.validator.constraints.Range.message            = muss zwischen {min} und {max} liegen
-org.hibernate.validator.constraints.SafeHtml.message         = k\u00F6nnte unsicheren HTML-Inhalt haben
-org.hibernate.validator.constraints.ScriptAssert.message     = Skriptausdruck "{script}" gab nicht true zur\u00FCck
-org.hibernate.validator.constraints.UniqueElements.message   = darf keine Duplikate enthalten
-org.hibernate.validator.constraints.URL.message              = muss eine g\u00FCltige URL sein
+org.hibernate.validator.constraints.CreditCardNumber.message        = ung\u00fcltige Kreditkartennummer
+org.hibernate.validator.constraints.Currency.message                = ung\u00fcltige W\u00e4hrung (muss eine der folgenden sein: {value})
+org.hibernate.validator.constraints.EAN.message                     = ung\u00fcltiger {type}-Barcode
+org.hibernate.validator.constraints.Email.message                   = keine korrekt formatierte E-Mail-Adresse.
+org.hibernate.validator.constraints.ISBN.message                    = Ung\u00fcltiger ISBN-Wert
+org.hibernate.validator.constraints.Length.message                  = L\u00e4nge muss zwischen {min} und {max} sein.
+org.hibernate.validator.constraints.CodePointLength.message         = L\u00e4nge muss zwischen {min} und {max} sein.
+org.hibernate.validator.constraints.LuhnCheck.message               = Die Pr\u00fcfziffer f\u00fcr ${validatedValue} ist ung\u00fcltig, Luhn Modulo 10-Kontrollsumme ist fehlgeschlagen.
+org.hibernate.validator.constraints.Mod10Check.message              = Die Pr\u00fcfziffer f\u00fcr ${validatedValue} ist ung\u00fcltig, Modulo 10-Kontrollsumme ist fehlgeschlagen.
+org.hibernate.validator.constraints.Mod11Check.message              = Die Pr\u00fcfziffer f\u00fcr ${validatedValue} ist ung\u00fcltig, Modulo 11-Kontrollsumme ist fehlgeschlagen.
+org.hibernate.validator.constraints.ModCheck.message                = Die Pr\u00fcfziffer f\u00fcr ${validatedValue} ist ung\u00fcltig, ${modType}-Kontrollsumme ist fehlgeschlagen.
```

Comment:
The case is not consistent with the other messages. I think it should be "die".

Reply:
It's related to the added "." This makes it a regular sentence, in which case the first word should start with an upper-case latter. But as said I'd not add the dot, in which case the "die" should be lower-cased.